### PR TITLE
Update Syphilis AOE copy and order

### LINKED ISSUE #8100: [Syphilis] Add condition and new AOE questions to Conduct Test - Test Card screen
### Description

Enhance the Test Card to incorporate new Ask on Entry (AOE) questions and symptoms related to a new disease, allowing users to select relevant information when adding a test result for a patient.

### Proposed solution

- **New AOE Questions Integration**: The Test Card should seamlessly integrate the new AOE questions, presenting them in a clear and user-friendly format.
- **Symptom Selection**: Provide a mechanism for users to select relevant symptoms from a predefined list.
- **Data Accuracy**: Ensure that the captured AOE responses and symptom selections are accurate and stored correctly in the database.
- **Responsiveness**: The Test Card should adapt to different screen sizes and devices, providing a consistent experience across platforms.

### Additional context
**Please note the new question asking if the patient has been told they have syphilis in the past. This question will be present for Syphilis only.*

**Mapping to FHIR:**
Has the patient been told they have syphilis before?
[Radio button] Yes: 1087151000119108
[Radio button] No: 14732006
[Radio button] Prefer not to answer: 261665006

**Copy change only**
**And the revised/update text for symptoms asking if they are experiencing or showing signs of symptoms. This can be updated to display with the same text for all conditions.** 
Is the patient currently experiencing or showing signs of symptoms?
[Radio button] Yes
[Radio button] No

--
From Nicole Gallaway, CDC SME for Syphilis:

> _There is no ‘verified list of symptoms’. I think the signs/symptom list you show looks generally good, except alopecia appears missing._ 

Suggested order:

```
Syphilis result
[Radio button] Positive
[Radio button] Negative
[Radio button] Inconclusive

What is the gender of their sexual partners? (Select all that apply)
[checkbox] Female
[checkbox] Male
[checkbox] Transwoman
[checkbox] Transman
[checkbox] Nonbinary or gender non-conforming
[checkbox] Gender identity not listed here
[checkbox] Prefer not to answer

Has the patient been told they have syphilis before?
[Radio button] Yes: 1087151000119108
[Radio button] No: 14732006
[Radio button] Prefer not to answer: 261665006

Is the patient pregnant?
[Radio button] Yes
[Radio button] No
[Radio button] Prefer not to say

Is the patient currently experiencing or showing signs of symptoms?
[Radio button] Yes
[Radio button] No

Which symptoms are they experiencing?
- Genital sore/lesion (SNOMED: 724386005)
- Anal sore/lesion (SNOMED: 195469007)
- Sore(s) in mouth/lips (SNOMED: 26284000)
- Body Rash (SNOMED: 266128007)
- Palmar (hand)/plantar (foot) rash (SNOMED: 56940005)
- Flat white warts (SNOMED: 91554004)
- Hearing loss (SNOMED: 15188001)
- Blurred vision (SNOMED: 246636008)
- Alopecia (SNOMED: 56317004)

Date of symptom onset
[MM/DD/YYYY]

[Button] Submit results
```

### PR DESCRIPTION
**Pull request by emyl3**
# FRONTEND PULL REQUEST

## Related Issue

- Resolves #8100 

## Changes Proposed

- This ticket became mostly a copy change and style update ticket since I think Bob(?) had already done all of the work to add syphilis AOE to the test card.
- Change "Is the patient currently experiencing any symptoms?" to "Is the patient currently experiencing or showing signs of symptoms?" for all diseases
- Fix incorrect SNOMED for "Blurred vision"
- Add "Alopecia" to symptoms
- Change order of AOE Syphilis questions as requested in the ticket
- Make style updates to Syphilis form to satisfy the following part of acceptance criteria:
> Responsiveness: The Results screen should adapt to different screen sizes and devices, providing a consistent experience across platforms.

## Additional Information
- I thought there would be more work for this ticket... Let me know if I'm missing any context from that ticket 😅 

## Testing
- Test locally and test Test Cards
   - see updated symptoms copy for every disease
   - try responsiveness of Syphilis form at different breakpoints
   - see updated symptoms for Syphilis
   
  
## Screenshots / Demos
**Before**
- Desktop
Screenshot 2024-09-20 at 10 24 17

- iPad
Screenshot 2024-09-20 at 10 25 05
Screenshot 2024-09-20 at 10 25 12

**After**
- Desktop
Screenshot 2024-09-20 at 10 26 22

- iPad
Screenshot 2024-09-20 at 10 26 37
Screenshot 2024-09-20 at 10 26 41

### CODE REVIEW

File: frontend/src/patientApp/timeOfTest/constants.ts [hunk 1]
```diff
@@ -97,8 +97,8 @@ export const syphilisSymptomsMap = {
   "56940005": "Palmar (hand)/plantar (foot) rash",
   "91554004": "Flat white warts",
   "15188001": "Hearing loss",
-  "46636008": "Blurred vision",
```
Comment (emyl3) [inline frontend/src/patientApp/timeOfTest/constants.ts:100]: ⚠️ Noticed the SNOMED we had originally for "Blurred vision" was incorrect (before 46636008, after [246636008](https://browser.ihtsdotools.org/?perspective=full&conceptId1=246636008&edition=MAIN/2023-09-01&release=&languages=en&latestRedirect=false)) see #8100 description as well

File: frontend/src/app/testQueue/testCardTestConstants.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ const GENDER_SEXUAL_PARTNERS_FEMALE_OVERRIDE = {
 };
 const BLURRED_VISION_OVERRIDE = {
   symptoms:
-    '{"15188001":false,"26284000":false,"46636008":true,"56940005":false,"68225006":false,"91554004":false,"195469007":false,"266128007":false,"724386005":false}',
+    '{"15188001":false,"26284000":false,"56317004":false,"56940005":false,"91554004":false,"195469007":false,"246636008":true,"266128007":false,"724386005":false}',
```
Comment (emyl3) [inline frontend/src/app/testQueue/testCardTestConstants.ts:86]: ⚠️ the change here is `Alopecia (SNOMED: 56317004)`, the snomed for blurred vision is corrected from `46636008` to `246636008` and the order of the others were changed

**Review by mpbrown — APPROVED** [review]
LGTM running it locally!

**Review by bobbywells52 — APPROVED** [review]
Tested locally and everything looks good to me. The styling on the tablets looks so much better 🤩 great work on this!

**Pull request merged**